kinder/ci/tools/update-workflows/templates/workflows/dryrun-tasks.yaml

```diff
@@ -60,6 +60,22 @@ tasks:
       docker exec {{ .vars.clusterName }}-control-plane-1 kubeadm init --ignore-preflight-errors={{ .vars.kubeadmIgnorePreflightErrors }} --kubernetes-version={{ .vars.kubernetesVersion }} --dry-run=true --v={{ .vars.kubeadmVerbosity }} || exit 1
       docker exec {{ .vars.clusterName }}-control-plane-1 kubeadm init --ignore-preflight-errors={{ .vars.kubeadmIgnorePreflightErrors }} --kubernetes-version={{ .vars.kubernetesVersion }} --upload-certs=true --dry-run=true --v={{ .vars.kubeadmVerbosity }} || exit 1
   timeout: 5m
+- name: join-dryrun-without-real-control-plane
+  description: |
+    Join the other nodes to the Kubernetes cluster in dryrun mode without a real control plane
+  cmd: /bin/bash
+  args:
+    - -c
+    - |
+      required_version="1.32" # join dry-run without real control plane is available since v1.32
+      current_version=$(echo "{{ .vars.kubernetesVersion }}" | grep -oP '\d+\.\d+')
+      if awk "BEGIN {exit !($current_version < $required_version)}"; then
+        echo "Join dry-run without real control plane is not available for Kubernetes version {{ .vars.kubernetesVersion }}. Skipping."
+        exit 0
+      fi
```

Comment:
i like this check. it avoids us having to write updates to the kinder config (new params) or adding more workflow files.